06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js

```diff
@@ -0,0 +1,53 @@
+class InvertedIndex {
+
+  constructor() {
+    this.textArray = [];
+    this.index = {};
+  };
+
+  transformToSingles(file) {

```

Comment:
Yes I have it and yes I'll include all that before Monday and hopefully by Tuesday, bossman Neji and I will be done scrutinising my work.